cfme/containers/provider/__init__.py

```diff
@@ -162,7 +162,10 @@ def num_service(self):
 
     @num_service.variant('ui')
     def num_service_ui(self):
-        return int(self.get_detail("Relationships", "Services"))
+        if version.current_version() < "5.7":
```

Comment:
If it were me, I'd do it this way to minimize code path

```python
        if version.current_version() < "5.7":
            name = "Services"
        else:
            name = "Container Services"
        return int(self.get_detail("Relationships", name))
```

but each to their own ;)